src/test/java/pageComponents/moderator/ModeratorOwnersTablePageComponent.java

```diff
@@ -1,18 +1,25 @@
 package pageComponents.moderator;
 
 import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.devtools.v85.page.Page;
 import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
 
 import java.util.List;
 
 public class ModeratorOwnersTablePageComponent {
 
+    private WebDriver driver;
+
     @FindBy(xpath = "//table/tbody/tr")
     private List<WebElement> owners;
 
     private WebElement neededOwner;
-    public ModeratorOwnersTablePageComponent(String ownerEmail)   {
+    public ModeratorOwnersTablePageComponent(WebDriver driver, String ownerEmail)   {
+        PageFactory.initElements(driver, this);
+        this.driver = driver;
         for(WebElement owner : owners)    {
```

Comment:
You should crate a separate method for this

Reply:
done